### ISSUE TITLE
Caching Question / Refresh From File Button?

### ISSUE THREAD

**Issue by ianc125** [Tenma-Server/Tenma#41]
When a comic is loaded into the reader and then the raw comic files are changed the changes dont show in the reader. It only shows the originally cached copy. I think if you had an option on the issue / series page to reload the issue/series from the files similar to what happens when you do the initial import. Similar to how Plex works.

**Comment by ianc125** [comment]
I think im off here, it looks like your reading from the raw file everytime anyways now that I've been through the code a bit. I was seeing an intermittent issue with the ComicRack.xml file and the reader that made it seem like it was caching or something. You can ignore this now.

**Comment by hmhrex** [comment]
No, you're correct. It caches it into a temp directory. I change the file names to be really short so I don't run into the "File path is too long" issue on Windows. I'm going to keep this one open. I can check for page amount pretty easily. If it doesn't match, the cache can be recreated. I'm not sure about how to find file changes though. I'll have to think on that one.

**Comment by ianc125** [comment]
In that case you could do a force refresh Issue/Series button similar to how Plex works for Movies. That would force a reload of the raw files from the Issue/Series back into the cache.

**Comment by hmhrex** [comment]
I like that! I'm planning on having a refresh button similar to Plex for the library. I'll see if I can fit this in that functionality.

**Comment by bpepple** [comment]
I've just about finished adding the ability to import files with ComicInfo.xml in my branch (tho I still need to add back the ability to search CV for files without the ComicInfo.xml), and one of the changes I've made is that when you import new files it will check the existing files in the db for any files that have had the modification date has changed.
